.github/workflows/style_check.yml

```diff
@@ -42,7 +43,22 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Check style with cpplint
```

Comment:
Could you please keep the style check for the C++ code?

You can create a file, say `python_style_check.yml`, for checking the python code.